modules/checkout/templates/commerce-checkout-order-summary.html.twig

```diff
@@ -0,0 +1,28 @@
+{#
+/**
+ * See commerce-order-receipt.html.twig for examples.
+ *
+ * Available variables:
+ * - order_entity: Order entity
+ * - items: The order items
+ * - totals: Result from commerce_order.order_total_summary service
+ * - checkout_flow_plugin_id: The current checkout flow ID.
```

Comment:
What's the point of checkout_flow_plugin_id and checkout_flow_step if we're not using them?